contracts/PangolinExchange.sol

```diff
@@ -26,16 +27,15 @@ contract PangolinExchange is Ownable, IAssetsExchange {
 
   /* ========= MODIFIERS ========= */
 
-  modifier RefundRemainder {
-    _;
+  function refundAvaxBalance() private nonReentrant {
```

Comment:
It's best to mark nonReentrant on funcions that could be invoked from external context. 
I'm not sure is marking private function nonReentract improves anything